DFC.Digital/DFC.Digital.Web.Sitefinity.JobProfileModule/Mvc/Views/JobProfileWhatYouWillDo/Index.cshtml

```diff
@@ -6,7 +6,7 @@
         <h2 class="heading-large">@Html.Raw(Model.Title)</h2>
         @if (Model.IsIntroActive && !string.IsNullOrWhiteSpace(Model.Introduction))
         {
-            <p>@Model.Introduction</p>
+            <p>@Html.Raw(Model.Introduction)</p>
```

Comment:
Can't we get the authors to apply wraping paragraph in the CMS? Alternatively, should this be a non HTML field?

Reply:
Discussed with BA and Design and this will be kept as a non HTML field for this release. Therefore closing this PR